setup.py

```diff
@@ -13,15 +13,19 @@
 
 #TODO add entry points for executables
 setup(name='enzy_htp',
-        version='0.1.0',
+        version='0.1.1',
```

Comment:
nice! advance!

Reply:
thanks! 🤣 Tried to get the pip bugs to go away